ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/ldesfragment/entities/LdesFragment.java

```diff
@@ -2,16 +2,21 @@
 
 import be.vlaanderen.informatievlaanderen.ldes.server.domain.ldesfragment.valueobjects.FragmentInfo;
 import be.vlaanderen.informatievlaanderen.ldes.server.domain.ldesfragmentrequest.valueobjects.FragmentPair;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class LdesFragment {
 
 	private final String fragmentId;
 
 	private final FragmentInfo fragmentInfo;
 
+	private static final Logger LOGGER = LoggerFactory.getLogger(LdesFragment.class);
+
 	public LdesFragment(final FragmentInfo fragmentInfo) {
 		this.fragmentInfo = fragmentInfo;
 		this.fragmentId = fragmentInfo.generateFragmentId();
+		LOGGER.debug("Fragment with id " + fragmentId + " created");
```

Comment:
Keep in mind this constructor doesn't only get called when creating a new LDES fragment. 
When converting a LdesFragmentEntity to an LdesFragment (happens when fetching fragments), this same constructor is used.

Reply:
This has been moved to the 3 fragmentCreator classes.